Crawler.py

```diff
@@ -50,7 +54,8 @@ def run(self):
                 f = open('data/' + event + '.html', 'w')
                 f.write(newContent)
                 f.close()
-
+                e_mail_message = f'Subject: Festival News!\n \nThere is a new change for the Event {event}, check of the website for tickets!'
```

Comment:
For now this is fine, when you extend this in the next step pass the event as a parameter of the `notify()` method, so that the default text can be in there instead of passing the whole thing from outside.